dist/docker/redhat/build_docker.sh

```diff
@@ -89,6 +89,8 @@ bcp LICENSE-ScyllaDB-Source-Available.md /licenses/
 run microdnf clean all
 run microdnf --setopt=tsflags=nodocs -y update
 run microdnf --setopt=tsflags=nodocs -y install hostname kmod procps-ng python3 python3-pip
+run curl -L --output /etc/yum.repos.d/scylla.repo https://downloads.scylladb.com/unstable/scylla/master/rpm/centos/latest/scylla.repo
```

Comment:
`unstable` means we'll need constant patching every time we branch, because it will refer to the wrong repo.

Better to consult SCYLLA-VERSION-GEN and construct the URL from the contents.

Reply:
Make sense, will do